bitcash/crypto.py

```diff
@@ -20,3 +23,110 @@ def ripemd160_sha256(bytestr):
 
 
 hash160 = ripemd160_sha256
+
+
+def sha512(bytestr):
+    return _sha512(bytestr).digest()
+
+
+# ECIES encryption/decryption methods; AES-128-CBC with PKCS7 is used
+# as the cipher; hmac-sha256 is used as the mac
+# Implementation follows the Elecrton-Cash implementaion of the same
```

Comment:
Some typos here :)

Reply:
Also `implementaion` -> `implementation` @yashasvi-ranawat

Reply:
woah, didn't see that :sweat_smile: